src/ui/Quickview/Quickview.ts

```diff
@@ -244,13 +245,36 @@ export class Quickview extends Component {
     // If there is no content inside the Quickview div,
     // we need to add something that will show up in the result template itself
     if (/^\s*$/.test(this.element.innerHTML)) {
+      const resultsSection = $$(this.root).find('.coveo-results-column');
       const iconForQuickview = $$('div', { className: 'coveo-icon-for-quickview' }, SVGIcons.icons.quickview);
       SVGDom.addClassToSVGInContainer(iconForQuickview.el, 'coveo-icon-for-quickview-svg');
       const captionForQuickview = $$('div', { className: 'coveo-caption-for-icon', tabindex: 0 }, 'Quickview'.toLocaleString()).el;
+      const captionForQuickviewArrow = $$('div').el;
+      captionForQuickview.appendChild(captionForQuickviewArrow);
       const div = $$('div');
       div.append(iconForQuickview.el);
       div.append(captionForQuickview);
       $$(this.element).append(div.el);
+
+      const captionForQuickviewPopper = new PopperJs(iconForQuickview.el, captionForQuickview, {
+        placement: 'bottom',
+        modifiers: {
+          preventOverflow: {
+            boundariesElement: resultsSection,
+            padding: 0
+          },
+          arrow: {
+            element: captionForQuickviewArrow
+          },
+          offset: {
+            offset: '0,8'
```

Comment:
Is it related to the CSS appearance of the arrow ?

If such, maybe a good idea to add a generic comment in the CSS that changing X value in the CSS will affect this part of the code ?
And vice versa here, a comment that specifies that if you change this, you should also check the CSS for X value.

Reply:
This is for pushing down the whole tooltip. Popper manages the `top` css property which is removed now in the css. But I could still add some comment